packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -2648,6 +2667,67 @@ describe.each([
           )
         }
       )
+
+      it.each(testScenarios)(
+        "does not enrich relationships when not enabled (via %s)",
+        async (__, retrieveDelegate) => {
+          await withCoreEnv(
+            {
+              TENANT_FEATURE_FLAGS: ``,
+            },
+            async () => {
+              const otherRows = _.sampleSize(auxData, 5)
+
+              const row = await config.api.row.save(tableId, {
+                title: generator.word(),
+                relWithNoSchema: [otherRows[0]],
+                relWithEmptySchema: [otherRows[1]],
+                relWithFullSchema: [otherRows[2]],
+                relWithHalfSchema: [otherRows[3]],
+                relWithIllegalSchema: [otherRows[4]],
+              })
+
+              const retrieved = await retrieveDelegate(row)
+
+              expect(retrieved).toEqual(
+                expect.objectContaining({
```

Comment:
Is this a correct way to assert what we want here? Won't this assertion also pass if the row _is_ enriched?

Reply:
Yes. `expect.objectContaining` does not act in nested properties. In this case we want to make sure that `relWithNoSchema` and any other field only have `_id` and `primaryDisplay`